Gemfile

```diff
@@ -22,6 +22,7 @@ gem "lodash-rails",                   "~>3.10.0"
 gem "momentjs-rails",                 "~>2.10.3"
 gem "patternfly-sass",                "~>2.2.0"
 gem "sass-rails"
+gem "high_voltage",                   "~>2.4.0"
```

Comment:
The rest of this list seems to be in alphabetical order.

Also, much as it's UI-related-ish, I'm not sure this actually counts as client-side anything.